dashboard/src/shared/utils.ts

```diff
@@ -137,6 +136,32 @@ export function getPluginPackageName(plugin?: Plugin | PluginNames | string) {
   }
 }
 
+// FIXME(agamez): replace with a proper call to the plugins server (see getPluginsServiceClientImpl)
```

Comment:
This is a workaround for setting the `Plugin` object later on. However, we still need a way to get (from the UI) the current plugins (closely related to #3889 )